src/NUnitTestAdapter/NUnitTestAdapter.cs

```diff
@@ -131,16 +134,43 @@ protected void Initialize(IDiscoveryContext context, IMessageLogger messageLogge
             {
                 Settings.Load(context);
                 TestLog.Verbosity = Settings.Verbosity;
-               
+                InitializeForbiddenFolders();
+                SetCurrentWorkingDirectory();
             }
             catch (Exception e)
             {
                 TestLog.Warning("Error initializing RunSettings. Default settings will be used");
                 TestLog.Warning(e.ToString());
             }
+            
+        }
+
+        public void InitializeForbiddenFolders()
+        {
+            ForbiddenFolders = new List<string>
```

Comment:
Could be shorter, `new[]` instead of `new List<string>` (not important)

Reply:
Done